numeros = [1,2,3,4,5,6,7,8,9,10]

 for numero in numeros:
print(numero)

#Esse erro ocorre, pois o phyton espera um bloco de identação e não encontra.